versions/3.0.4.md

```diff
@@ -383,7 +382,7 @@ An object representing a Server.
 
 | Field Name | Type | Description |
 | ---- | :----: | ---- |
-| <a name="server-url"></a>url | `string` | **REQUIRED**. A URL to the target host. This URL supports Server Variables and MAY be relative, to indicate that the host location is relative to the location where the OpenAPI document is being served. Variable substitutions will be made when a variable is named in `{`braces`}`. |
+| <a name="server-url"></a>url | `string` | **REQUIRED**. A URL to the target host. This URL supports Server Variables and MAY be relative, to indicate that the host location is relative to the location where the entry document of the OpenAPI Description is being served. Variable substitutions will be made when a variable is named in `{`braces`}`. |
```

Comment:
```suggestion
| <a name="server-url"></a>url | `string` | **REQUIRED**. A URL to the target host. This URL supports Server Variables and MAY be relative, to indicate that the host location is relative to the location where the OpenAPI document is being served. Variable substitutions will be made when a variable is named in `{`braces`}`. |
```

Retaining small-"d" because the document in question may or may not be a proper OpenAPI Document.

Reply:
Not sure whether the above suggested change is still proposed or now withdrawn.

I prefer the text

> relative to the location where the entry document of the OpenAPI Description is being served

Reply:
Not sure whether the above suggested change is still proposed or now withdrawn.

I prefer the text

> relative to the location where the entry document of the OpenAPI Description is being served

Reply:
@ralfhandl that makes sense to me and I'd support it.  I think it would have bo be a new suggestion?  How about this, which is more explicit about which "document" without tying it to a format:

```suggestion
| <a name="server-url"></a>url | `string` | **REQUIRED**. A URL to the target host. This URL supports Server Variables and MAY be relative, to indicate that the host location is relative to the location where the document containing the Server Object is being served. Variable substitutions will be made when a variable is named in `{`braces`}`. |
```

Reply:
Excellent!

Reply:
I'm fine with this wording so I am going accept this suggestion.